src/requirements/reqs-functions.ts

```diff
@@ -94,32 +73,27 @@ function iterateThroughUniversityRequirements(
     minCount: 0,
     applies: 'all'
   } as const;
```

Comment:
What does `as const` do in this case?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions
It's necessary to make TypeScript type `fulfilledBy` as an enum rather than a regular string.